sway/desktop/wl_shell.c

```diff
@@ -47,10 +47,6 @@ static void set_position(struct sway_view *view, double ox, double oy) {
 	view->swayc->y = oy;
 }
 
-static void set_activated(struct sway_view *view, bool activated) {
-	// no way to activate wl_shell
-}
-
 static void close(struct sway_view *view) {
```

Comment:
Can we not call these functions `close`? That name conflicts with another commonly used function.